certified-connectors/DomainToolsIrisInvestigate/apiDefinition.swagger.json

```diff
@@ -1593,8 +1644,8 @@
                                                         "count": {
                                                             "type": "integer",
                                                             "format": "int32",
-                                                            "description": "Count",
-                                                            "title": "Count"
+                                                            "description": "Count of connected domains sharing ip address",

```

Comment:
@RamboV if you use abbreviations, please make sure to capitalize them. You can refer to [English strings guidelines](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings).